docs/DafnyRef/UserGuide.md

```diff
@@ -361,9 +368,11 @@ The command emits exit codes of
 - 2 for parsing, type-checking or serious errors in running the auditor (e.g. failure to write a report or when report comparison fails)
 - 0 for normal operation, including operation that identifies audit findings
 
+It also takes the `--verbose` option, which then gives information about the files being formatted.
+
 #### 13.5.1.9. `dafny format` {#sec-dafny-format}
 
-Dafny supports a formatter, which for now only changes the indentation of every line in a Dafny file, so that it conforms
+Dafny supports a formatter, which for now only changes the indentation lines in a Dafny file, so that it conforms
```

Comment:
I prefer the original wording but it's not a blocker. Could we write "indentation of lines in..."?

Reply:
That is what I meant. Mikael's words say that every line is altered, which is not the case.